src/cdn_definitions/data.yaml

```diff
@@ -94,6 +94,21 @@ origin_alias:
   dest: /origin/rpms
 
 
+# Aliases between two RHEL-X repositories, both of which contain the latest RHEL-X release.
+releasever_alias:
+
+# Each item defines a "src" (a floating RHEL-X repository that always contains the latest content
+# for a major RHEL release) and a "dest" (a RHEL-X.Y repository, containing the published content
+# for the latest minor release version of RHEL-X).
+#
+# For example, /content/dist/rhelX/X should be an alias for /content/dist/rhelX/X.Y, where X is a
+# major version of RHEL and X.Y is a minor version of RHEL.
```

Comment:
If we're not intending to start using this for the versions in rhel_open_dict, it could also be helpful for this comment to point out something like:

> These aliases are intended for use with product versions no longer receiving releases, i.e. those
> not covered by rhel_open_dist.  For currently active versions, aliases are instead derived from
> other fields such as env_to_releasever_mappings.